lib/users.js

```diff
@@ -96,7 +96,7 @@ export async function removeUser({accessManager, user}) {
 }
 
 export function createOnboardLink({user, profile}) {
-  let onboardLink = `${window.location.origin}/onboard`;
+  let onboardLink = 'https://localhost:38443/onboard';
```

Comment:
This should probably be moved to the config, as well.

Reply:
Moved [2a23f2b](https://github.com/digitalbazaar/bedrock-web-wallet/pull/4/commits/2a23f2bffc02e7f03509a98428538c43e3af2c65)